src/page.njk

```diff
@@ -42,4 +42,14 @@ permalink: "/{{ '/' if pageItem.slug === 'home' else pageItem.slug  }}/"
 </article>
 {% endif %}
 {% endblock %}
-{{ content | safe }}
+
+{% block footerScripts %}
+<script src="/js/content-header-observer.js"></script>
+<script>
+	// Track all h1 and h2 in content for the About and Inclusion Challenges page.
+document.querySelectorAll("main article.page h1, main article.page h2, header").forEach((section) => {
```

Comment:
Why is `header` elements tracked?

Reply:
So when the header is in the top 6% of the viewport the side menu will stop highlighting items. See line `17` of `content-header-observer.js`

Reply:
I see. Thanks for the explanation. Could you add a code comment at the line 17 of `content-header-observer.js` with this explanation? Thanks.